.s2i/httpd-cfg/01-commercial.conf

```
@@ -382,6 +387,13 @@ AddType text/vtt                            vtt
     RewriteRule ^pipelines/?$ /pipelines/latest [R=302]
     RewriteRule ^pipelines/latest/?(.*)$ /pipelines/1\.15/$1 [NE,R=302]
 
+    # Pipelines release notes version switch
+
+    RewriteRule ^pipelines/1\.(10|11|12|13|14|15)/release_notes/op-release-notes-1-([0-9]+)\.html$ pipelines/1.$1/about/op-release-notes.html [L,R=302]
+    RewriteRule ^pipelines/1\.(16|17|17)/release_notes/op-release-notes?(.*)\.html$ pipelines/1.$1/release_notes/pipelines-release-notes.html [L,R=302]
+    RewriteRule ^pipelines/1\.(16|17|17)/anput/op-release-notes\.html$ pipelines/1.$1/release_notes/pipelines-release-notes.html [L,R=302]
```

```suggestion
    RewriteRule ^pipelines/1\.(16|17|18)/release_notes/op-release-notes?(.*)\.html$ pipelines/1.$1/release_notes/pipelines-release-notes.html [L,R=302]
    RewriteRule ^pipelines/1\.(16|17|18)/anput/op-release-notes\.html$ pipelines/1.$1/release_notes/pipelines-release-notes.html [L,R=302]
```
Looks like you have 17 twice here. Not sure if it's supposed to be 18 or something else.